include/sot/core/parameter-server.hh

```diff
@@ -118,7 +118,7 @@ public:
       DYNAMIC_GRAPH_ENTITY_WARNING(*this)
           << "Cannot get parameter " << ParameterName
           << " before initialization!\n";
-      Type ParameterValue;
+      Type ParameterValue=Type();
```

Comment:
Same remark as https://github.com/stack-of-tasks/sot-core/pull/162/files#r489217847